README.md

```diff
@@ -73,51 +149,53 @@ For example:
 [{
   emailMd5Lower: "1105677c8d9decfa1e36a73ff5fb5531",
   emailSha1Lower: "ba9d46a037766855efca2730031bfc5db095c654"
-}, {
-  emailMd5Lower: "9e3669d19b675bd57058fd4664205d2a",
-  emailSha1Lower: "e9d71f5ee7c92d6dc9e92ffdad17b8bd49418f98"
 }]
 ```
 
-## Consuming campaign requests
 
-We create a campaign request when an advertiser wants to send a campaign to one of your subscribers.
 
-There are two ways to consume campaign requests:
+<br />
 
- 1. [Real time](#real-time-campaign-request-listener).
- 2. [Batch](#batch-campaign-requests).
- 
-### Real-time campaign-request listener
+## Configuring DNS
 
-To receive campaign requests in real-time, create an HTTPS listener and <a href="mailto:Traverse Operations <operations@traversedlp.com&gt">let us know the URL</a>.
+Traverse uses Amazon SES to send email. You will need to make a few minor DNS updates on your end to get things started.
 
-We will POST JSON objects with the following properties to the URL:
+1. __Choose a subdomain to mail from.__
 
-| Property | Description |
-|----------|-------------|
-| `campaignId` | Campaign ID |
-| `emailMd5Lower` | MD5 hash of trimmed, lowercased email address |
-| `emailSha1Lower` | MD5 hash of trimmed, lowercased email address |
-| `advertiserProperties` | Custom advertiser properties (*advanced*) |
+    This can be any subdomain you are not currently using (i.e. `{subdomain}.{publisher-domain}.com`).
 
-For example:
+2. __Configure the subdomain to receive email.__
 
-```json
-{
-  "campaignId": "6a11644c-690d-4bf3-bb19-4c3efba5a5a5",
-  "emailMd5Lower": "1105677c8d9decfa1e36a73ff5fb5531",
-  "emailSha1Lower": "ba9d46a037766855efca2730031bfc5db095c654",
-  "listIds": ["772823bd-b7be-4d23-bd78-96a577d02765"],
-  "advertiserProperties": {
-    "impressionId": "f53f6078-f802-4c98-90ca-e90aa56995ab",
-    "foo": "bar"
-  }
-}
-```
+    In order to legally send email, Traverse must be able to receive incoming messages to this subdomain. To facilitate this, set up a CNAME record which points the sending subdomain domain to our receiving domain. The record should look like this:
 
-*Note:* If the listener does not promptly reply with an HTTP 2xx status code, we may resubmit the request, later.
+    | Record Name | Record Type | Record Value |
+    ------|-------|-------|
+    | `{subdomain}.{publisher-domain}.com` | CNAME | srvrt.com |
 
-### Batch campaign requests
+3. __Configure the subdomain to send email.__
 
-*Batch campaign requests are not yet supported.*
+    The final step is to configure your subdomain to demonstrate ownership and configure DKIM. This is handled by configuring a number of TXT records for the domain. These settings are generated
+    on a per-domain process, so <a href="mailto:Traverse Operations <operations@traversedlp.com&gt">contact us</a> when you are ready to make the changes.
+
+
+<br />
+
+## Publisher Header and Footer
```

Comment:
My impression is that this documentation would be implementation details. There's some overlap with what would be in the "Publisher Field Guide". The other pieces are implementation details. This part is obviously important, but I'm not sure if it is really implementation details.